### DISCUSSION TITLE
TypeError Issue

### DISCUSSION THREAD

**Discussion by mrmorgan-ai** [TomSchimansky/CustomTkinter#1258] [General]
Hi folks, can someone help me with this result? i got this error:TypeError: CTkEntry.__init__() missing 1 required positional argument: 'master' after i run this line:  prompt = ctk.CTkEntry(height=40, width=512,text_font=("Arial",20),text_color="black",fg_color="white")
Any ideas of how i could fix that? pleaase. its been several days

**Comment by Wolf-SO** [comment]
Yes, **All** widgets (apart from `CTk` ans `CTkToplevel` take at least one parameter on creation: the parent widget. This is the case in all GUI toolkits I'm aware of. BTW: have allok on [Creating and highlighting code blocks - GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks "Creating and highlighting code blocks - GitHub Docs") for formatting code boxes.
